docs/sdk/client-side-sdks/flutter/flutter-gettingstarted.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Flutter SDK Getting Started
+sidebar_label: Getting Started
+sidebar_position: 2
+---
+
+[![Pub](https://img.shields.io/pub/v/devcycle_flutter_client_sdk)](https://img.shields.io/pub/v/devcycle_flutter_client_sdk)
```

Comment:
Hmm! Interesting choice. I hadn't put much thought myself into where we put these. I'm ultimately not sure what the best approach is.
I think in the others so far, we only have them in the top level page, so in this case it would just be the flutter.md file, rather than at the top of each of them.